model/validation.go

```diff
@@ -931,14 +934,18 @@ func ValidateProxyMeshConfig(mesh *proxyconfig.ProxyMeshConfig) (errs error) {
 		}
 	}
 
-	if err := ValidatePort(int(mesh.ProxyListenPort)); err != nil {
+	if err := ValidatePort(int(mesh.ProxyListenPort), true); err != nil {
 		errs = multierror.Append(errs, multierror.Prefix(err, "invalid proxy listen port:"))
 	}
 
-	if err := ValidatePort(int(mesh.ProxyAdminPort)); err != nil {
+	if err := ValidatePort(int(mesh.ProxyAdminPort), true); err != nil {
```

Comment:
Making the admin port optional seems unrelated feature. 
Making the proxy port optional also seems extra complexity we don't really need, and makes testing matrix and configuration more complicated.

Can we leave move 'make ports optional' to separate PR ? 
( in particular since we must have at least one port )